zio-crypto/shared/src/test/scala/zio/crypto/encryption/HybridEncryptionSpec.scala

```diff
@@ -1,57 +1,57 @@
 package zio.crypto.encryption
 
-import java.nio.charset.StandardCharsets.US_ASCII
-
+import zio.Scope
 import zio.crypto.keyset.KeysetManager
-import zio.test.Assertion._
 import zio.test._
 
-object HybridEncryptionSpec extends DefaultRunnableSpec {
+import java.nio.charset.StandardCharsets.US_ASCII
+
+object HybridEncryptionSpec extends ZIOSpecDefault {
 
   private def testAlgorithm(algorithm: HybridEncryptionAlgorithm) = suite(algorithm.toString)(
     suite("bytes")(
-      testM("encrypt(m, k) != encrypt(m, k)") {
-        checkM(Gen.chunkOf(Gen.anyByte)) { m =>
+      test("encrypt(m, k) != encrypt(m, k)") {
+        check(Gen.chunkOf(Gen.byte)) { m =>
           for {
             key         <- KeysetManager.generateNewAsymmetric(algorithm)
             ciphertext1 <- HybridEncryption.encrypt(m, key.publicKeyset)
             ciphertext2 <- HybridEncryption.encrypt(m, key.publicKeyset)
-          } yield assert(ciphertext1)(not(equalTo(ciphertext2)))
+          } yield assertTrue(!(ciphertext1 == ciphertext2))
```

Comment:
I think `a != b` is easier to read :)

Reply:
💯 I agree. I was mostly just following the auto-suggestion from the zio plugin while I was fixing these specs. I'll update these too.